### PR TITLE
PRS in BLorient3716b (owners and translators)

### PR DESCRIPTION
**Pull request by CarstenHoffmannMarburg**

### CODE REVIEW

File: new/PRS14541PeterBrru.xml [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+        </profileDesc>
+        <revisionDesc>
+            <change who="CH" when="2024-08-27">Created entity</change>
+        </revisionDesc>
+    </teiHeader>
+    <text>
+        <body>
+            <listPerson>
+                <person sex="1">
+                    <persName xml:lang="en" xml:id="n1">Peter Brru</persName>
```
Comment (eu-genia) [inline new/PRS14541PeterBrru.xml:48]: here https://archive.org/details/in.ernet.dli.2015.503297/page/n187/mode/2up  "Peter Bru" is mentioned as a translator who (with his son) accompanied the British mission of Wylde in 1883

Comment (CarstenHoffmannMarburg) [reply new/PRS14541PeterBrru.xml:48]: I tried to create a wikidata-lemma for him, but all I got was L1347920. I think, this is wrong. What shall I do to create a person with Q..... ?

https://www.wikidata.org/wiki/Lexeme:L1347920

Comment (CarstenHoffmannMarburg) [reply new/PRS14541PeterBrru.xml]: I cannot figure it out from this document, but I think, it is Wylde, Augustus Blandy (ca.1850-1909) (PRS10176Wylde) as one of the prominent members of the Wylde family: https://reed.dur.ac.uk/xtf/view?docId=ark/32150_s1j6731385f.xml

Comment (eu-genia) [reply new/PRS14541PeterBrru.xml]: wikidata creation https://www.wikidata.org/w/index.php?title=Special:NewItem&label=petros%20birru

As for Wylde, what I suggested was also to add the bibliographic reference to the source. Clearly it is Augustus Blandy Wylde, there is an entry about him in EAe and his name/initials are provided in the source quoted, it is easy to find the book references and the book cover

Augustus Blandy Wylde, '83 to '87 in the Soudan: With an Account of Sir William Hewett's Mission to King John of Abyssinia, I, London: Remington and Co, 1888, p. 180

**Review by eu-genia — APPROVED** [review]

**Review by eu-genia — APPROVED** [review]

**Pull request merged**